stage0/run.go

```diff
@@ -379,6 +382,27 @@ func preparedWithPrivateUsers(dir string) (string, error) {
 	return string(bytes), nil
 }
 
+func addResolvConf(cfg RunConfig, rootfs string) {
+	content := "# Generated by rkt\n\n"
+	if len(cfg.DNSSearch) > 0 {
+		content += fmt.Sprintf("search %s\n", strings.Join(cfg.DNSSearch, " "))
+	}
+	for _, server := range cfg.DNS {
+		if server == "" {
+			continue
+		}
+		content += "nameserver " + server + "\n"
+	}
+	if len(cfg.DNSOpt) > 0 {
+		content += fmt.Sprintf("options %s\n", strings.Join(cfg.DNSOpt, " "))
```

Comment:
But rkt would need to keep the list of options in sync with the glibc options when new options are added. It does not happen that often but still. The last one to be added seems to be `use-vc` documented in man-pages in 2015.